crates/core/src/models/felt.rs

```diff
@@ -44,10 +46,10 @@ impl From<H160> for Felt252Wrapper {
     }
 }
 
-impl From<Felt252Wrapper> for H256 {
+impl From<Felt252Wrapper> for H160 {
     fn from(felt: Felt252Wrapper) -> Self {
         let felt: FieldElement = felt.into();
-        H256::from_slice(&felt.to_bytes_be())
+        H160::from_slice(&felt.to_bytes_be()[12..])
```

Comment:
do we really want this? when do we troncate a felt into a H160?
only when we troncate starknet_address to eth address?

Reply:
I guess what Im trying to say is:
either we have a TryFrom that doesn't troncate, but fails if felt is too big, or we have a troncate, but it must be clear a dev's head that Felt to h160 can loose information

Reply:
I think this was added by @jobez, it is flagged as a change I did because I reorganized the file a bit. We could either change to `TryFrom` but exposing ourself to a bunch of failed conversions, or we troncate only if the slice doesn't fit.

Reply:
let's try to ask him what is the goal we want to achieve and choose accordingly between the options you offer